airflow/executors/base_executor.py

```diff
@@ -69,6 +69,8 @@ class BaseExecutor(LoggingMixin):
     job_id: None | int | str = None
     callback_sink: BaseCallbackSink | None = None
 
+    is_local: bool = False
```

Comment:
We should add a simple unit test that asserts that this field is present with the default that we expect (and likewise add one for the specific executors which override it to ensure we get the expected value there). If someone were to accidentally (or misguidedly change it with intention) modify one of these we'd like tests to fail.

Reply:
Good idea, i'll do that :)

Reply:
Done :)